### PR TITLE
some-minor-changes

### PR DESCRIPTION
**Pull request by Jianghai-Quantil**
@nibinqtl Please review.
Correct the unit for origin_fast_route
Mention unit for origin_limit_rate
Mention that $request_time is the same as $down_time

### CODE REVIEW

File: en/docs/edge-logic/built-in-variables.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ If you use the CDN Pro portal to edit the property, typing `$` at a position whe
...
-| <span id="download_time"></span><span class="var">$download_time</span> |<span class="badge small" title="numerical value">#</span> the time from when CDN Pro receives the client request until CDN Pro sends the response body to the system buffer, the value should be equal to sum of $request_end_time/$turn_around_time/$transfer_time, in seconds. Another variable download_time_ms shares the same meaning but in milliseconds.  | <span class="badge yellow">No</span> | <span class="badge green">Yes</span>
+| <span id="download_time"></span><span class="var">$download_time</span> |<span class="badge small" title="numerical value">#</span> the time from when CDN Pro receives the client request until CDN Pro sends the response body to the system buffer, the value should be equal to sum of $request_end_time/$turn_around_time/$transfer_time, in seconds. Another variable download_time_ms shares the same meaning but in milliseconds. The meaning of $download_time is the same as another variable $request_time. | <span class="badge yellow">No</span> | <span class="badge green">Yes</span>
```
Comment (burtonquantil) [inline en/docs/edge-logic/built-in-variables.md:25]: I cannot comment on technical correctness, but I'd suggest some minor English improvements.

the time in seconds from when CDN Pro receives the client request until CDN Pro sends the response body to the system buffer. The value should be equal to the sum of $request_end_time/$turn_around_time/$transfer_time.
Another variable, download_time_ms, shares the same meaning but is measured in milliseconds. The meaning of $download_time is the same as another variable, $request_time.

File: en/docs/edge-logic/supported-directives.md [hunk 2]
```diff
@@ -461,7 +461,7 @@ This is an enhancement of the [proxy_connect_timeout](http://nginx.org/en/docs/h
 **Default:** `origin_fast_route off;` <br/>
 **Context:** server, location, if in location
 
-This directive enables a fast route to be used to access the origin. It is powered by our proprietary HDT technology which provides more reliable connection with reduced latency. The traffic transferred through this fast route may be charged with a higher rate than the edge traffic. The traffic such transferred is subject to a rate limit of 3 Mbps. Please contact our customer support if a higher limit is required.
+This directive enables a fast route to be used to access the origin. It is powered by our proprietary HDT technology which provides more reliable connection with reduced latency. The traffic transferred through this fast route may be charged with a higher rate than the edge traffic. The traffic such transferred is subject to a rate limit of 3 MByte/s. Please contact our customer support if a higher limit is required.
```
Comment (burtonquantil) [inline en/docs/edge-logic/supported-directives.md:464]: correction to the English of two sentences.

 The traffic transferred through this fast route may be charged **at** a higher rate than the edge traffic.  The traffic is restricted to a rate of 3 MByte/s.

File: en/docs/edge-logic/supported-directives.md [hunk 3]
```diff
@@ -520,7 +520,7 @@ Although CDN Pro has a hierarchical cache structure, the directive changes the h
 **Default:** `origin_limit_rate 30m;`<br>
 **Context:** server, location
 
-This is a wrapper of the [proxy_limit_rate](http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_limit_rate) directive. It limits the speed at which the response is read from the origin server.
+This is a wrapper of the [proxy_limit_rate](http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_limit_rate) directive. It limits the speed at which the response is read from the origin server. Measured in in bytes/sec.
```
Comment (burtonquantil) [inline en/docs/edge-logic/supported-directives.md:523]: remove one "in" from the least phrase.

**Review by burtonquantil — CHANGES_REQUESTED** [review]
I suggested some corrections to the English  text

**Comment by Jianghai-Quantil** [comment]
@burtonquantil Revised per your suggestions. Please review again.

**Review by burtonquantil — APPROVED** [review]
The English version looks fine to merge.
Thanks for the updates.

**Pull request merged**